src/main/java/com/sandvoxel/generitech/common/tileentities/power/TestPower.java

```diff
@@ -314,4 +107,13 @@ public boolean hasCapability(Capability<?> capability, EnumFacing facing) {
 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested code block into a method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1199)